sdk/src/main/java/io/radar/sdk/model/RadarUser.kt

```diff
@@ -119,9 +119,20 @@ class RadarUser(
     /**
      * The user's current trip.
      */
-    val trip: RadarTrip?
+    val trip: RadarTrip?,
+
+    /**
+     * A boolean indicating whether the user's IP address is being mocked, such as in a simulation. May be `false` if
+     * Fraud is not enabled.
+     */
+    val mocked: Boolean? = null
```

Comment:
nit: this should be typed the same as `proxy`, right? L117 above just has `: Boolean`

Reply:
No - I purposely am allowing this to be nullable in order to have a better mechanism for making the RadarFraud object null. Also since this is a new object with a default value, kotlin implementations of the SDK won't need to do anything if any implementations call the RadarUser constructor anywhere.